content/about/branding.en.md

```diff
@@ -0,0 +1,143 @@
+---
+title: branding
+displayTitle: Branding of Node.js
+description: 'Branding | Node.js'
+authors: AugustinMauroy
```

Comment:
I think it's unfair to remove the previous authors, as they also contributed to part of this file.

Reply:
This is an oversight because I recreated an empty file

Reply:
it's add 🎉!